README.md

```diff
@@ -1,13 +1,17 @@
 # embedded-common
 This repository contains the Sensirion i2c abstraction layer and commonly shared
 code. It is a dependency of the Sensirion embedded drivers and it should not be
-necessary to clone it independently.
+necessary to clone it independently, as we recommend to download the repository
+recursively from the respective projects.
 
-We recommend to download the repository recursively within the
-[SCD](https://github.com/Sensirion/embedded-scd),
-[SGP](https://github.com/Sensirion/embedded-sgp),
-[SHT](https://github.com/Sensirion/embedded-sht) or
-[SPS](https://github.com/Sensirion/embedded-sps) projects.
+If you're looking for the drivers, please refer to the release bundles (zip
+files) from the respective projects:
+
+[SCD](https://github.com/Sensirion/embedded-scd/releases)
+[SGP](https://github.com/Sensirion/embedded-sgp/releases)
+[SHT](https://github.com/Sensirion/embedded-sht/releases)
+[SPS](https://github.com/Sensirion/embedded-sps/releases)
+[STS](https://github.com/Sensirion/embedded-sts/releases)
```

Comment:
I would either keep the commas and "or" or use a bullet list.

Reply:
Should we list the UART repos as well?

Reply:
Yup. Better? https://github.com/Sensirion/embedded-common/blob/af3f0ce731a64f93b6f3bc53d3d9d636d36f2c42/README.md